ApeWorX/ape
# docs: add documentation to the 'ape.managers' classes

### What I did

Doc strs for the managers!

### How I did it

Lots of revising.

### How to verify it

build the docs!

You should see stuff like this.

Review the words of the docs.

<img width="301" alt="Screen Shot 2021-12-15 at 16 14 29" src="https://user-images.githubusercontent.com/19540978/146273317-040e89e9-f77c-4107-8585-20f40bfe7d0c.png">

<img width="1102" alt="Screen Shot 2021-12-15 at 16 14 36" src="https://user-images.githubusercontent.com/19540978/146273282-10b0f560-da3e-4bac-8ae7-fd78c1fd2bed.png">

### Checklist
